app/main.py

```diff
@@ -1 +1,41 @@
-#  write your code here
+class Animal:
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True
+                 ) -> None:
```

Comment:
Place all arguments including `self` on a new line. You should not have any arguments on the same line as the opening parenthesis since your closing parenthesis does not have any arguments on the same line.